include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -191,7 +191,7 @@ __radix_sort_count_submit(_ExecutionPolicy&& __exec, ::std::size_t __segments, :
         oneapi::dpl::__ranges::all_view<_CountT, __par_backend_hetero::access_mode::read_write>(__count_buf);
 
     // submit to compute arrays with local count values
-    sycl::event __count_levent = __exec.queue().submit([&](sycl::handler& __hdl) {
+    return __exec.queue().submit([&](sycl::handler& __hdl) {
```

Comment:
I personally think this change makes the code less readable, pulling the return up above the kernel definition.

Reply:
ok, restored.